aptos-move/block-executor/src/lib.rs

```diff
@@ -33,7 +33,7 @@ describes the updates made by the incarnation as (memory location, value) pairs.
 The write-set of the incarnation is applied to shared memory (the multi-version
 data-structure) at the end of execution. After an incarnation executes it needs
 to pass validation. The validation re-reads the read-set and compares the
-observed versions. Intuitively, a successful validation implies that writes
+observed versions. Intuitively, a successful validation implies that reads
```

Comment:
Both `reads` and `writes` are up-to-date. Here saying `writes` may be more consistent with rest of the comments.